commons/Cargo.toml

```
@@ -23,7 +27,12 @@ tempfile = "3"
 thiserror = "1"
 walkdir = "2"
 which_problem = "0.1"
+ascii_table = { version = "4.0.2", features = ["color_codes"] }
+const_format = "0.2.31"
```

(For consistency with the dependency version strategy used already)

```suggestion
ascii_table = { version = "4", features = ["color_codes"] }
const_format = "0.2"
```